tests/periph_eeprom/main.c

```diff
@@ -98,17 +99,58 @@ static int cmd_write(int argc, char **argv)
     return 0;
 }
 
+static int cmd_test(int argc, char **argv)
+{
+    (void)argc;
+    (void)argv;
+    
+    uint8_t bytes[256];
+    eeprom_clear(0, 256);
```

Comment:
This function doesn't exist (so I don't think will ever build) but it's interesting to have it (something I have in mind already). Are you willing to propose this in another PR ?

Reply:
Yep, we have it in our code and forget to remove it here. Actually it's just eeprom_write with all zeros.